packages/components/src/systems/DraftPrescriptions/index.tsx

```diff
@@ -193,8 +206,8 @@ export default function DraftPrescriptions(props: DraftPrescriptionsProps) {
     setIsLoading(false);
   }
 
-  onMount(() => {
-    if (merged.templateIds.length > 0 || merged.prescriptionIds.length > 0) {
+  createEffect(() => {
```

Comment:
any issue we can imagine here since this was in an onMount rather than a createEffect